web/template/create-draft.gohtml

```diff
@@ -216,6 +216,32 @@
             <input class="govuk-input {{ if .Error.Field.donorEmail }}govuk-input--error{{ end }}" id="f-donorEmail" name="donorEmail" value="{{ .Form.Email }}" />
           </div>
 
+          <div class="govuk-form-group {{ if .Error.Field.types }}govuk-form-group--error{{ end }}" id="f-correspondenceTypes">
+            <fieldset class="govuk-fieldset" aria-describedby="f-correspondence-types-hint">
+              <legend class="govuk-fieldset__legend">Does the donor require correspondence in Welsh or large format?</legend>
+              <div class="govuk-hint" id="f-correspondence-types-hint">Optional, select all that apply</div>
+              {{ template "errors" .Error.Field.types }}
```

Comment:
I think if an error occurred it would be on the `correspondenceByWelsh` or `correspondenceLargeFormat` field, not `types`. It's unlikely that an error _would_ occur, but probably worth including both here so they'd show up somewhere:
```suggestion
              {{ template "errors" .Error.Field.correspondenceByWelsh }}
              {{ template "errors" .Error.Field.correspondenceLargeFormat }}
```
You'll need to change `{{ if .Error.Field.types }}govuk-form-group--error{{ end }}` above to check these fields too.

Reply:
Ahh good spot!